docs/pim/create_custom_name_schema_strategy.md

```diff
@@ -0,0 +1,27 @@
+---
+description: Create custom name schema strategy to generate URL aliases based on attribute values.
+---
+
+# Create custom name schema strategy
+
+Make sure the attributes are configured correctly. 
```

Comment:
Why are we doing it? What problem are we solving or what improvement implementing?